### ISSUE TITLE
StackOnlyBar Option Not Working in Blazor Version of ApexCharts

### ISSUE THREAD

**Issue by paulowrld** [apexcharts/Blazor-ApexCharts#539]
I am encountering an issue with the `StackOnlyBar` option in the Blazor version of ApexCharts. Although `StackOnlyBar` was implemented and adjusted in the JavaScript version (v4.0), it does not seem to work in the Blazor-ApexCharts library. Specifically, when attempting to create a mixed chart with two stacked bar series and one separate line series, the bars are not stacking as intended.

**To Reproduce**

Here is a simplified version of my Blazor component demonstrating the issue:

```razor
@page "/test2"

@using ApexCharts
@using System.Globalization

<h3>StackOnlyBar Chart Example</h3>

<ApexChart TItem="ChartData" Height="400" Options="chartOptions">
    <!-- Stacked Bar Series 1 -->
    <ApexPointSeries TItem="ChartData"
                     Items="ChartDataList"
                     SeriesType="@SeriesType.Bar"
                     Name="Completed"
                     Color="#28B463"
                     XValue="@(item => item.MonthYear)"
                     YValue="@(item => (decimal)item.Completed)" />

    <!-- Stacked Bar Series 2 -->
    <ApexPointSeries TItem="ChartData"
                     Items="ChartDataList"
                     SeriesType="@SeriesType.Bar"
                     Name="Pending"
                     Color="#F1C40F"
                     XValue="@(item => item.MonthYear)"
                     YValue="@(item => (decimal)item.Pending)" />

    <!-- Line Series -->
    <ApexPointSeries TItem="ChartData"
                     Items="ChartDataList"
                     SeriesType="@SeriesType.Line"
                     Name="Target"
                     Color="#FF5733"
                     XValue="@(item => item.MonthYear)"
                     YValue="@(item => (decimal)item.Target)" 
                     Stroke="@(new ApexCharts.SeriesStroke { Width = 3 })" />
</ApexChart>

@code {
    // Data Model
    public class ChartData
    {
        public string MonthYear     { get; set; } = string.Empty;
        public double Target        { get; set; }
        public double Completed     { get; set; }
        public double Pending       { get; set; }
    }

    // Chart Options
    private ApexChartOptions<ChartData> chartOptions;

    // Sample Data
    private List<ChartData> ChartDataList = new List<ChartData>
    {
        new ChartData { MonthYear = "Jan/24", Target = 5000, Completed = 3000, Pending = 2000 },
        new ChartData { MonthYear = "Feb/24", Target = 7000, Completed = 4000, Pending = 3000 },
        new ChartData { MonthYear = "Mar/24", Target = 6000, Completed = 4500, Pending = 1500 },
        new ChartData { MonthYear = "Apr/24", Target = 8000, Completed = 5000, Pending = 3000 },
        new ChartData { MonthYear = "May/24", Target = 7500, Completed = 6000, Pending = 1500 },
        new ChartData { MonthYear = "Jun/24", Target = 9000, Completed = 7000, Pending = 2000 },
        new ChartData { MonthYear = "Jul/24", Target = 8500, Completed = 6500, Pending = 2000 },
        new ChartData { MonthYear = "Aug/24", Target = 9500, Completed = 8000, Pending = 1500 },
        new ChartData { MonthYear = "Sep/24", Target = 10000, Completed = 8500, Pending = 1500 },
        new ChartData { MonthYear = "Oct/24", Target = 11000, Completed = 9000, Pending = 2000 },
        new ChartData { MonthYear = "Nov/24", Target = 10500, Completed = 9500, Pending = 1000 },
        new ChartData { MonthYear = "Dec/24", Target = 12000, Completed = 10000, Pending = 2000 }
    };

    protected override void OnInitialized()
    {
        chartOptions = new ApexChartOptions<ChartData>
        {
            Chart = new()
            {
                Toolbar = new ApexCharts.Toolbar
                {
                    Tools = new Tools { Zoom = false, Zoomin = false, Zoomout = false, Pan = false, Reset = false, Download = false },
                },
                DropShadow = new ApexCharts.DropShadow
                {
                    Enabled = true,
                    Color = "#000",
                    Top = 5,
                    Left = 0,
                    Blur = 3,
                    Opacity = 0.1
                },
                StackOnlyBar = true,
                //Type = ChartType.Line
                //Stacked = true
            },
            Grid = new Grid
            {
                BorderColor = "#e7e7e7",
                StrokeDashArray = 3
            },
            Tooltip = new ApexCharts.Tooltip
            {
                X = new TooltipX
                {

                }
            },
            Stroke = new Stroke { Curve = Curve.Smooth },
            Yaxis = new List<YAxis>()
            {
                new YAxis
                {
                    Labels = new YAxisLabels
                    {
                        Formatter = @"function (value) {
                        return 'R$' + Number(value).toLocaleString();
                        }"
                    },
                }
            },
        };
    }
}
```

**Expected behavior**

I expect the two bar series ("Completed" and "Pending") to be stacked on top of each other for each month, while the line series ("Target") remains separate. This should visually represent the completed and pending amounts stacked together with the target value overlaid as a line.

**Actual behavior**

The `StackOnlyBar` option does not seem to have any effect. The bar series ("Completed" and "Pending") are displayed side by side instead of being stacked, and the line series ("Target") overlays them without any stacking functionality.

Could you please provide an example or guidance on how to implement a mixed chart in Blazor with two stacked bar series and one separate line series using `StackOnlyBar`? An example would greatly help in understanding the correct configuration needed to achieve the stacking functionality in Blazor-ApexCharts.

Thank you for your assistance!

**Comment by joadan** [comment]
Hi can you provide a codepen using apexcharts.js to show what you are trying to do?
If it can be done in apexcharts.js we should be able to to it in the wrapper.

**Comment by paulowrld** [comment]
Hi

I'm trying to do something like this example: 
https://codepen.io/paulowrld999/pen/RwXzqKj

From what I understand, there used to be a bug that made this difficult, but now it's possible. However, I'm not sure how to implement it in Blazor instead of JavaScript

**Comment by joadan** [comment]
Thank you! I'll have a look..

**Comment by joadan** [comment]
Hi, I added a sample https://apexcharts.github.io/Blazor-ApexCharts/issues/539
The only change here is that both Stacked and StackOnlyBar is set to true.

**Comment by paulowrld** [comment]
Hi, I understand now.
I went back to check my code (not the example I sent, but another one I’m using) and identified the reason behind one of the issues I was having. If I use the property below with two values, for some reason, the line chart doesn’t render. Even in the example, if you modify it to include this code, it won’t work, even if another line is added.
`Stroke = new Stroke { Curve = [Curve.Straight, Curve.Smooth], },`
I remember trying to use both properties set to true, as you suggested, but it didn’t work. That’s why I found it surprising that the solution seemed so simple. So, I dug a bit deeper and discovered this issue.

In any case, thank you very much!
